index.html

```diff
@@ -6,6 +6,392 @@
 	<link rel="stylesheet" type="text/css" href="index.css">
 </head>
 <body>
-	
+
+	<!-- intro -->
+	<header class="row">
+
+		<img src="assets/yellow-pill.svg" alt="yellow pill">
+		<img src="assets/more-pills.svg" alt="a blue pill, a purple pill, and a yellow pill">
+		
+		<span class="subTitle">A Viget Exploration</span>
+		<h1>
+			<img src="assets/title.svg" alt="Improving Medical Adherence">	
+		</h1>
+		
+		<img src="assets/pills.svg" alt="a blue pill and yellow pill">
+		<img src="assets/purple-pills.svg" alt="two purple pills">
+
+		<!-- yellow-pill.svg is top left -->
+		<!-- more-pills.svg is top right -->
+		<!-- pills.svg is bottom left -->
+		<!-- purple-pills.svg is bottom right -->
+
+		<section class="whiteBox">
```

Comment:
Hey Cyron,
This specific section isn't too big to give a great example, but the main purpose* of BEM is to help describe the content within by using descriptive names that describe the content instead of how you might style them.
ex: Imagine a menu with 'li's inside.
If you were to use BEM (Block-Element_Modifier):
The menu would be the 'block', and the 'element' would be the 'li'. So if you were naming your menu it would be ```<nav class="menu">```, but then if you were naming your li (which you might not need to unless you wanted to use javascript to target them) they would be ```<li class="menu-item">```.

One thing I didn't mention was the modifier (the 'm' of BEM). This would be something like: ```<li class="menu-item_disabled">```. The modifier describes something happening on an element that goes against the basic styling. Hopefully this gives you a starting point for naming conventions. 

Here's the link to BEM (don't get too caught up with all the other acronyms listed): http://getbem.com/introduction/

I'm still figuring out how to properly do this as well. It's the weekend, so I don't think you'll get a reply back from Megan until next week. Let me know if I can help in any way!